cmd/kuberhealthy/reflector.go

```diff
@@ -74,6 +74,22 @@ func (sr *StateReflector) CurrentStatus() health.State {
 
 		log.Debugln("Getting status of check for web request to status page:", khState.GetName(), khState.GetNamespace())
 
+		// if there is a requested namespace, then filter out checks from other namespaces
+		if len(namespace) != 0 {
+			if khState.GetNamespace() != namespace {
+				log.Debugln("Skipping", khState.GetName(), "because it is not from the", namespace, "namespace")
+				continue
+			}
+		}
+
+		// if there is no requested namespace, show checks from "kube-system" and "kuberhealthy"
```

Comment:
this is a preference to us as the project owners, but is an unlikely assumption for all other users.  I would make the default "blank" functionality pick everything.